src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    
     public String getRandomColor() {
-        return null;
+        int randomBallNumber = new Random().nextInt(Color.values().length);
+        return Color.values()[randomBallNumber].toString();
```

Comment:
Use name() for getting String representation of enum constants
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)